FirebaseCore/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
   XCFrameworks.
 - Update underlying FIRLogger implementation from `asl` to `os_log`.
 - Remove `FIRLoggerForceSTDERR` configuration option.
+- [changed] Move `Timestamp` class into `FirebaseCore`. `FirebaseFirestore.Timestamp`
+  was changed to `FirebaseCore.Timestamp`. (#13221)
```

Comment:
> `FirebaseFirestore.Timestamp` was changed to `FirebaseCore.Timestamp`.

@paulb777 – This is a small break, but I believe we could add:
```swift
@_exported import class FirebaseCore.Timestamp
```
to selectively import and re-export that class so it stills appears under the `FirebaseFirestore` namespace (like what was done with Swift extensions post-consolidation)

Reply:
SGTM. It should minimize the break.

Reply:
Should I create a new PR to add this line in `FirebaseFirestoreSwift.swift` or there is a pr working in progress?

Reply:
@cherylEnkidu, sure! I haven't started anything. For all the files in this PR where you needed to add `import FirebaseCore`, I _think_ you should be able to just replace those with `@_exported import class FirebaseCore.Timestamp`.

Reply:
Got it, I will send you a PR shortly.